Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +13,207 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string[] postTimes, string[] showTimes, string expected)
-        {
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
             var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+        }
+
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
+            int[] previousStart = null;
+            int[] previousEnd = null;
+
+            for (int i = 0; i < exactPostTime.Length; i++) {
+                DateTime prevDateStart = DateTime.Now,

```

Comment:
This is really hard to understand, given the names of the variables and that too, so many. See if you can use methods to modularize the code. Put proper comments explaining which part of the code is doing what. May be add comments against each variable explaining it's usage.